src/assets/stylesheets/custom.scss

```diff
@@ -20,9 +21,12 @@ $font11: 11px;
 $font12: 12px;
 $font13: 13px;
 $font14: 14px;
+$font15: 15px;
 $font16: 16px;
 $font18: 18px;
 $font20: 20px;
```

Comment:
curiosity on all of the font variables there are here:

- PatternFly has a default set of variables related to the 12px default size. They also have a set of variables that are relationship based, so that this method of multiple variables for increasing font sizes by 1px is not needed.
- Ideally, if custom variables are required for all of these fonts outside of the scope of PatternFly, then they would be increased using em or rem, not pixels.

PatternFly custom variables: https://github.com/patternfly/patternfly/blob/master/src/less/variables.less